### ISSUE TITLE
Remove BPM references in the created scripts

### ISSUE THREAD

**Issue by tqvarnst** [eschabell/jboss-demo-template#1]
Currently the jboss-demo-template generates an example project with BPM, but not a template. Both has values, but I rather see the template clean and without BPM specific references.

If this is aiming to be a template generation script for multiple JBoss demos it shouldn't use specific references to BPM Suite. 

For example: 
1. The ASCII header in init.[sh|bat] states BPM SUITE. I recommend replacing BPM SUITE with JBOSS DEMO.
2. The generated init.[sh|bat] contains variables like BPMS=jboss-bpms-installer-6.0.3.GA-redhat-1.jar. I recommend that we use neutral variables like for example BINARY

or

For multi product demos this could be a list or binaries that the demo depends on. For example:

```
    BINARIES=(jboss-product1.zip jboss-product2.zip) #Place any downloads here within () and with a space between them
    ...
    for DONWLOAD in ${BINARIES[@]}
    do
       if [[ -r $SRC_DIR/$DONWLOAD || -L $SRC_DIR/$DONWLOAD ]]; then
        echo $DONWLOAD are present...
        echo
       else
        echo You need to download $DONWLOAD from the Customer Support Portal or jboss.org/products
        echo and place it in the installs directory to proceed...
        echo
        exit
       fi
    done
```
1. Remove BPM specific instructions like in the generated init.[sh|bat], like: 
   
   ```
   java -jar $SRC_DIR/$BPMS $SUPPORT_DIR/installation-bpms -variablefile $SUPPORT_DIR/installation-bpms.variables
   ```
2.  Remove demo specific configs from the generated init.[sh|bat] like:
   
   ```
   cp -r $PRJ_DIR/acme-demo-flight-service/target/acme-flight-service-1.0.war $SERVER_DIR
   ```

**Comment by eschabell** [comment]
Good idea, submit a PR please!

**Comment by tqvarnst** [comment]
Next time Eric, please make sure that you take owner ship of a issue before fixing it. I cannot push my fix for this since you have been fixing this too.

**Comment by eschabell** [comment]
https://github.com/jbossdemocentral/jboss-demo-template/commit/3a0092bdca7ad2249a1a7d9f57ca14b1c6ab27c9 fixes #1.
